core/src/test/java/org/elasticsearch/bootstrap/BootstrapForTesting.java

```diff
@@ -68,11 +68,12 @@
         }
 
         // just like bootstrap, initialize natives, then SM
-        Bootstrap.initializeNatives(javaTmpDir, true, true, true);
+        boolean seccomp = Boolean.parseBoolean(System.getProperty("tests.seccomp", "true"));
```

Comment:
can this use a non-lenient boolean parser? it shoudl be hard, like really hard, to turn this off. not the other way around.